master/buildbot/test/integration/test_integration_secrets_with_vault.py

```diff
@@ -30,22 +30,31 @@
 class SecretsConfig(RunMasterBase):
 
     def setUp(self):
-        rv = os.system("docker run -d -e SKIP_SETCAP=yes -e "
-                       "'VAULT_DEV_ROOT_TOKEN_ID=my_vaulttoken' -e 'VAULT_TOKEN=my_vaulttoken'"
-                       " --name=vault_for_buildbot -p 8200:8200 vault")
-        if rv != 0:
+        try:
```

Comment:
I guess this is unrelated.

Reply:
Completely unrelated, this is why I put it in a separate commit

Reply:
The problem is that I missed it in my first overlook as the patch is so huge..